### ISSUE TITLE
Write a 0.6.0 release announcement for the website and forum

### ISSUE THREAD

**Issue by akien-mga** [OpenDungeons/OpenDungeons#846]
#845 will likely be a prerequisite so that we know what to put the focus on.

@Bertram25 is usually pretty good at this exercise. *Evil grin*

**Comment by Bertram25** [comment]
> @Bertram25 is usually pretty good at this exercise. _Evil grin_

eh eh.

**Comment by Bertram25** [comment]
@akien-mga I'll happily take that one if you do the release notes. Deal?

**Comment by akien-mga** [comment]
Deal :)

**Comment by Bertram25** [comment]
Will be done as a first draft tonight. :) @akien-mga will be able to fix my mistakes then, and add his own stuff. ;)

**Comment by Bertram25** [comment]
First draft!! http://forum.freegamedev.net/viewtopic.php?f=35&t=6454

**Comment by Bertram25** [comment]
I'll make a PR for the website part when you think the announcement is ready. I guess we'll need a few more pictures...

**Comment by akien-mga** [comment]
For the website, I was wondering if we should not do a "next" branch or something, and do this kind of PR in this new branch. This way we can merge the PR, and continue working on the new content, without pushing it live on the website.

Use case that I have in mind: merge your current announcement which is quite cool ASAP, and then let me make a second PR to adjust some formatting/punctuation for better legibility. WDYT?

tl;dr: There are a couple things I would like to fix in the announcement, but I'm too lazy to do some quoting in the forum, so I'd prefer to work directly on the HTML version ;)

**Comment by akien-mga** [comment]
As for the screenshots, I'll have a look. I guess we can put a couple screenshots from far away to show a neat dungeon, and then close ups on new features: one or two spells, doors, tileset, etc.

**Comment by Bertram25** [comment]
> For the website, I was wondering if we should not do a "next" branch or something, and do this kind of PR in this new branch. This way we can merge the PR, and continue working on the new content, without pushing it live on the website.
> Use case that I have in mind: merge your current announcement which is quite cool ASAP, and then let me make a second PR to adjust some formatting/punctuation for better legibility. WDYT?

Yeah, good idea. :)

> As for the screenshots, I'll have a look. I guess we can put a couple screenshots from far away to show a neat dungeon, and then close ups on new features: one or two spells, doors, tileset, etc.

Yep. Sounds good to me.

**Comment by akien-mga** [comment]
I'll try to get started with the website part based on the content @Bertram25 wrote.

**Comment by akien-mga** [comment]
> I'll try to get started with the website part based on the content @Bertram25 wrote.

Done, the initial version by @Bertram25 + some adjustments is ready for merge into the news-0.6.0 branch: https://github.com/OpenDungeons/opendungeons.github.io/pull/14

I'll get to work and try to get some screenshots to illustrate the announcement.

**Comment by akien-mga** [comment]
Based on the current release announcement, I'd say I need to show at least:
- Particle effects (maybe one spell + a wizard firing)
- New research UI
- Doors
- Brick walls
- Creature mood icons
- Ruins of the Confluent "presqu'île" in no FOW mode

**Comment by akien-mga** [comment]
I added some screenshots here: https://github.com/OpenDungeons/opendungeons.github.io/pull/15

**Comment by akien-mga** [comment]
I guess this is now fixed.
